template/data/achievementinfo.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../../bower_components/iron-ajax/iron-ajax.html">
+
+<dom-module id="achievement-info">
+  <template>
+    <iron-ajax
+      id="ajax"
+      auto
+      url="achievementlist.json"
+      handle-as="json"
+      last-response="{{achievementinfo}}"
+      on-response="filter">
+    </iron-ajax>
+  </template>
+  <script>
+    Polymer({
+      is: 'achievement-info',
+
+      filter: function(e) {
+        this.achievementinfo = e.detail.response[this.achievementid]
+      },
+
+      _updateAchievementInfo: function() {
+        this.$.ajax.generateRequest()
```

Comment:
Debounce time is set as an attribute on iron-ajax: debounce-duration="300" (debounce duration will be 300ms here), or imperatively on the element as this.$.ajax.debounceDuration = ... If I remember right, 300ms is about the time a ¨standard user¨should be able to notice UI changes and react to it, hence the value "defaults" to this.
The documentation says that debounce is only used when generating requests automatically (auto attribute)... This means we might have to debounce ourselves using lodash or underscore libraries: this.debouncedAjaxCall = _.debounce(**function with logic for ajax call**, 300); Using lodash/underscore also gives us the opportunity between triggering the function on the leading or trailing edge (Example with lodash: https://lodash.com/docs/3.10.1#debounce).